components/crud-web-apps/tensorboards/frontend/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ng": "ng",
     "start": "npm run copyLibAssets && ng serve --base-href /tensorboard/ --deploy-url /tensorboard/",
-    "build": "npm run copyLibAssets && ng build --prod --base-href /tensorboard/ --deploy-url /tensorboard/static/",
+    "build": "npm run copyLibAssets && ng build --prod --base-href /tensorboards/ --deploy-url static/",
```

Comment:
In the line above the base-href is `/tensorboard/` instead of `/tensorboards/` which it is here.

Reply:
I'm not sure I understand what the question or suggested change is here. Could you rephrase this?

Reply:
There is an inconsistency in the usage of `--base-href /tensorboard/ --deploy-url /tensorboard/"` on Line 6 and `--base-href /tensorboards/` on Line 7.

Reply:
I see what you mean. The `start` script should be completely removed, since the users should run the UI locally with `build:watch` for now.

I'll push a commit to remove this script

Reply:
Added a commit that removes the unused npm script